Sources/Gui/Main.cpp

```diff
@@ -425,10 +425,8 @@ int main(int argc, char **argv) {
 						SDL_RWops *io = SDL_RWFromFile(
 						  (home + "/.openspades/CONTENT_MOVED_TO_NEW_DIR").c_str(), "wb");
 						if (io != NULL) {
-							const char *text = ("Content of this directory moved to " +
-							                    xdg_data_home + "/openspades")
-							                     .c_str();
-							io->write(io, text, strlen(text), 1);
+							std::string text = ("Content of this directory moved to " + directory);
```

Comment:
Build is failing because `directory` is not defined.

Reply:
... I need to stop writing these at 1 am. Sorry for the rookie mistake, should be fixed now!